app/views/MainTempleteNew.scala.html

```diff
@@ -73,7 +74,7 @@
         isWelshTranslationAvailable: Boolean = false,
         signOutUrl: Option[String] = None,
         serviceUrl: Option[String] = None,
-        backLinkUrl: Option[String] = None,
+        backLinkUrl: Option[String] = Some("javascript:history.go(-1)"),
```

Comment:
the backLinkContent doesn't come from a message

Reply:
@sergi-hmrc When I call this in TaxCodeComparrisonView I pass in the message key this is just a safety default. I do agree it should be changed, maybe we could raise a ticket to change this to an option[String]? We cant add the message in here in here I dont believe

@Jamie22Butler will discuss with UX what they wan the URL to be

Reply:
There has been a separate ticket raised https://jira.tools.tax.service.gov.uk/browse/PAYEE-298
 to address the backlink issues
For now I have changed the link to go back to whatDoYouWantToDoPage